terraform/postgresql/bind/versions.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    postgresql = {
```

Comment:
Are we still using this?

Reply:
Yes, it's needed for the transition to the new provider.